client/state/rewind/retention/actions.ts

```diff
@@ -0,0 +1,18 @@
+import { Action } from 'redux';
+import { JETPACK_BACKUP_RETENTION_UPDATE } from 'calypso/state/action-types';
+import type { RetentionPeriod } from './types';
+import 'calypso/state/data-layer/wpcom/sites/rewind/retention';
+
```

Comment:
This action creators file is missing:

```
import 'calypso/state/rewind/init';
```

which is necessary to ensure that when running an action creator, the corresponding dynamic reducer has already been loaded.